app/controllers/reviews_controller.rb

```diff
@@ -1,6 +1,12 @@
 class ReviewsController < ApplicationController
     def index
-        reviews = Review.ApplicationController
+        reviews = Review.all
         render json: reviews
     end
+
+    def show
+        @review = Review.find(params[:id])
+    end
```

Comment:
No need for a show method in your controller since you are performing this logic from the frontend once you have all of your reviews stored to state.